mods/other/enderpearl/init.lua

```diff
@@ -9,24 +9,21 @@ local callbacks = {}
 ----------------------
 
 minetest.register_craftitem("enderpearl:ender_pearl", {
-  description = S("Ender Pearl").."\n"..
-     S("Left click to throw it@nIt will teleport you on the node it hit making you 15 damage@n(it won't work if you launch it in an unloaded world area)"),
+  description = "It will teleport you on the node it hit for the cost of 15 HP point",
```

Comment:
If you decided to not use the translator anymore, then you may also remove the locale folder and minetest.get_translator() line.

Reply:
Ok